### DISCUSSION TITLE
每个字都认识但就是不懂的英文句子

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#27] [General]
# 每个字都认识但就是不懂的英文句子

收集的英文句子，这些句子很简单，每个字都看得懂但就是不明它的意思。

https://www.edony.ink/twitter/english-sentence-that-is-too-simple-to-understand/

**Comment by edonyzpc** [comment]
如果有英文大神私藏了有意思的句子欢迎留言，我会更新到文档里面去。
